src/transformers/models/paligemma/processing_paligemma.py

```diff
@@ -72,6 +74,18 @@ def build_string_from_input(prompt, bos_token, image_seq_len, image_token):
     """
     return f"{image_token * image_seq_len}{bos_token}{prompt}\n"
 
+class PaliGemmaProcessorKwargs(ProcessingKwargs, total=False):
+    _defaults = {
+        "text_kwargs": {
+            "tokenize_newline_separately": True,
+            "suffix": None,
+        },
+        "image_kwargs": {
+            "do_convert_rgb": None,
+            "do_thumbnail": None,
+            "do_align_long_axis":  None,
+        },
```

Comment:
Same here, not all kwargs from call method are in defaults and might need PaligemmaImageKwargs if not all of them are in default ImageKwargs